site/en/release_notes.md

```diff
@@ -6,6 +6,108 @@ summary: Milvus Release Notes
+- Fixed the panic caused by incorrect logic of getting unindexed segments ([#24061](https://github.com/milvus-io/milvus/pull/24061)).
+- Fixed the panic that occurs when a collection does not exist in quota effect ([#24321](https://github.com/milvus-io/milvus/pull/24321)).
+- Fixed an issue where refresh may be notified as finished early ([#24438](https://github.com/milvus-io/milvus/pull/24438)) ([#24466](https://github.com/milvus-io/milvus/pull/24466)).
+
+### Enhancement
+
+- Added an error response to return when an unimplemented request is received ([#24546](https://github.com/milvus-io/milvus/pull/24546))
+- Reduced disk usage for Milvus Lite and Standalone:
+
+  - Refine RocksDB option ([#24394](https://github.com/milvus-io/milvus/pull/24394))
+  - Fix RocksMQ retention not triggering at DataCoord timetick channel ([#24134](https://github.com/milvus-io/milvus/pull/24134))
+
+- Optimized quota to avoid OOM on search
+- Added consistency_level in search/query request ([#24541](https://github.com/milvus-io/milvus/pull/24541))
+- (pr24562) Supported search with default parameters ([#24516](https://github.com/milvus-io/milvus/pull/24516))
+- Put DataNode load statslog lazy if SkipBFStatsLog is true ([#23779](https://github.com/milvus-io/milvus/pull/23779))
+- Put QueryNode lazy load statslog if SkipBFLoad is true ([#23904](https://github.com/milvus-io/milvus/pull/23904))
+- Fixed concurrent map read/write in rate limiter ([#23957](https://github.com/milvus-io/milvus/pull/23957))
+- Improved load/release performance:
+
+  - Implemented more frequent CollectionObserver checks to trigger during load procedure ([#23925](https://github.com/milvus-io/milvus/pull/23925))
+  - Implemented checks to trigger while waiting for collection/partition to be released ([#24535](https://github.com/milvus-io/milvus/pull/24535))
+
+- Optimized PrivilegeAll permission check ([#23972](https://github.com/milvus-io/milvus/pull/23972))
+- Fixed the "not shard leader" error when gracefully stopping ([#24038](https://github.com/milvus-io/milvus/pull/24038))
+- Checked the overflow for inserted integer ([#24142](https://github.com/milvus-io/milvus/pull/24142)) ([#24172](https://github.com/milvus-io/milvus/pull/24172))
+- Lowered the task merge cap to mitigate an insufficient memory error ([#24233](https://github.com/milvus-io/milvus/pull/24233))
+- Removed constraint that prevents creating an index after load ([#24415](https://github.com/milvus-io/milvus/pull/24415))
+- Removed index check to trigger compaction ([#23657](https://github.com/milvus-io/milvus/pull/23657)) ([#23688](https://github.com/milvus-io/milvus/pull/23688))
+
+### Performance improvements
+
+- Added SIMD support for several filtering expressions ([#23715](https://github.com/milvus-io/milvus/pull/23715), [#23781](https://github.com/milvus-io/milvus/pull/23781)).
```

Comment:
add one more item  [Enhancement]: Optimize the search performance with high filtering ratio #23948